renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -7570,6 +7571,379 @@ void WrappedVulkan::vkCmdEndRendering(VkCommandBuffer commandBuffer)
+      ObjDisp(commandBuffer)->CmdCopyAccelerationStructureKHR(Unwrap(commandBuffer), &unwrappedInfo));
+
+  if(IsActiveCapturing(m_State))
+  {
+    {
+      CACHE_THREAD_SERIALISER();
+
+      SCOPED_SERIALISE_CHUNK(VulkanChunk::vkCmdCopyAccelerationStructureKHR);
+      Serialise_vkCmdCopyAccelerationStructureKHR(ser, commandBuffer, pInfo);
+
+      m_FrameCaptureRecord->AddChunk(scope.Get());
+    }
+
+    GetResourceManager()->MarkResourceFrameReferenced(GetResID(pInfo->src), eFrameRef_Read);
+    GetResourceManager()->MarkResourceFrameReferenced(GetResID(pInfo->dst), eFrameRef_CompleteWrite);
+  }
+}
+
+template <typename SerialiserType>
+bool WrappedVulkan::Serialise_vkCmdCopyAccelerationStructureToMemoryKHR(
+    SerialiserType &ser, VkCommandBuffer commandBuffer,
+    const VkCopyAccelerationStructureToMemoryInfoKHR *pInfo)
+{
+  SERIALISE_ELEMENT(commandBuffer);
+  SERIALISE_ELEMENT_LOCAL(Info, *pInfo);
+
+  SERIALISE_CHECK_READ_ERRORS();
+
+  if(IsReplayingAndReading())
+  {
+    VkCopyAccelerationStructureToMemoryInfoKHR unwrappedInfo = Info;
+    unwrappedInfo.src = Unwrap(unwrappedInfo.src);
+
+    ObjDisp(commandBuffer)->CmdCopyAccelerationStructureToMemoryKHR(Unwrap(commandBuffer), &unwrappedInfo);
+  }
+
+  return true;
+}
+
+void WrappedVulkan::vkCmdCopyAccelerationStructureToMemoryKHR(
+    VkCommandBuffer commandBuffer, const VkCopyAccelerationStructureToMemoryInfoKHR *pInfo)
+{
+  VkCopyAccelerationStructureToMemoryInfoKHR unwrappedInfo = *pInfo;
+  unwrappedInfo.src = Unwrap(unwrappedInfo.src);
+  SERIALISE_TIME_CALL(
+      ObjDisp(commandBuffer)
+          ->CmdCopyAccelerationStructureToMemoryKHR(Unwrap(commandBuffer), &unwrappedInfo));
+
+  if(IsActiveCapturing(m_State))
+  {
+    {
+      CACHE_THREAD_SERIALISER();
+
+      SCOPED_SERIALISE_CHUNK(VulkanChunk::vkCmdCopyAccelerationStructureToMemoryKHR);
+      Serialise_vkCmdCopyAccelerationStructureToMemoryKHR(ser, commandBuffer, pInfo);
+
+      m_FrameCaptureRecord->AddChunk(scope.Get());
+    }
+
+    GetResourceManager()->MarkResourceFrameReferenced(GetResID(pInfo->src), eFrameRef_Read);
+  }
+}
+
+template <typename SerialiserType>
+bool WrappedVulkan::Serialise_vkCmdCopyMemoryToAccelerationStructureKHR(
+    SerialiserType &ser, VkCommandBuffer commandBuffer,
+    const VkCopyMemoryToAccelerationStructureInfoKHR *pInfo)
+{
+  SERIALISE_ELEMENT(commandBuffer);
+  SERIALISE_ELEMENT_LOCAL(Info, *pInfo);
+
+  SERIALISE_CHECK_READ_ERRORS();
+
+  if(IsReplayingAndReading())
+  {
+    VkCopyMemoryToAccelerationStructureInfoKHR unwrappedInfo = Info;
+    unwrappedInfo.dst = Unwrap(unwrappedInfo.dst);
+
+    ObjDisp(commandBuffer)->CmdCopyMemoryToAccelerationStructureKHR(Unwrap(commandBuffer), &unwrappedInfo);
+  }
+
+  return true;
+}
+
+void WrappedVulkan::vkCmdCopyMemoryToAccelerationStructureKHR(
+    VkCommandBuffer commandBuffer, const VkCopyMemoryToAccelerationStructureInfoKHR *pInfo)
+{
+  VkCopyMemoryToAccelerationStructureInfoKHR unwrappedInfo = *pInfo;
```

Comment:
This function is going to be illegal to ever call (since we will always report ASs as incompatible) so I think this should be stubbed out rather than implemented.